modules/dxp/apps/commerce/commerce-dashboard-web/src/main/java/com/liferay/commerce/dashboard/web/internal/display/context/CommerceDashboardForecastDisplayContext.java

```diff
@@ -39,11 +46,54 @@ public CommerceDashboardForecastDisplayContext(
 			ConfigurationProviderUtil.getPortletInstanceConfiguration(
 				CommerceDashboardForecastPortletInstanceConfiguration.class,
 				_commerceDashboardForecastRequestHelper.getThemeDisplay());
+
+		_themeDisplay =
+			_commerceDashboardForecastRequestHelper.getThemeDisplay();
```

Comment:
I think you don't need it. You can get the same info from `_commerceDashboardForecastRequestHelper`